ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -8,16 +8,53 @@ import { isEqualCaseInsensitive } from '../../helpers/utils/util';
 
 export default function ConfirmAddSuggestedToken(props) {
   const {
-    history,
-    acceptWatchAsset,
-    rejectWatchAsset,
-    mostRecentOverviewPage,
     suggestedAssets,
     tokens,
+    rejectWatchAsset,
+    history,
+    mostRecentOverviewPage,
+    acceptWatchAsset,
```

Comment:
no logic changes in this commit. Just moved functions around. 

and accidentally reordered these props during a rebase